### PR TITLE
fix: Add regex check for db version

### PR DESCRIPTION
**Pull request by venky-wandb**

### CODE REVIEW

File: modules/database/main.tf [hunk 1]
```diff
@@ -11,7 +11,7 @@ resource "random_string" "master_password" {
 }
 
 locals {
-  is_mysql_80            = var.engine_version == "8.0.mysql_aurora.3.01.0"
+  is_mysql_80            = length(regexall("^8.0", var.engine_version)) > 0 ? true : false
```
Comment (venky-wandb) [inline modules/database/main.tf:14]: Per the [docs here](https://www.terraform.io/language/functions/regexall)

Comment (jsbroks) [reply modules/database/main.tf:14]: `is_mysql_80            = length(regexall("^8.0", var.engine_version)) > 0`

Comment (venky-wandb) [reply modules/database/main.tf:14]: Oh right, I don't need the explicit check 🤦‍♂️

**Review by jsbroks — APPROVED** [review]

**Comment by jsbroks** [comment]
This PR is included in version 1.5.3 :tada:

**Pull request merged**